src/observablemixin.js

```diff
@@ -414,6 +416,40 @@ function bindTo( ...args ) {
 	} );
 }
 
+// Binds to an attribute in a set of iterable observables.
+//
+// @private
+// @param {Iterable.<Observable>} observables
+// @param {String} attribute
+// @param {Function} callback
+function bindToMany( observables, attribute, callback ) {
+	if ( this._bindings.size > 1 ) {
+		/**
+		 * Binding one attribute to many observables only possible with one attribute.
+		 *
+		 * @error observable-bind-to-many-not-one-binding
+		 */
+		throw new CKEditorError( 'observable-bind-to-many-not-one-binding: Cannot bind multiple properties with toMany().' );
+	}
+
+	this.to(
+		// Bind to #attribute of each observable...
+		...getBindingTargets( observables, attribute ),
+		// ...using given callback to parse attribute values.
+		callback
+	);
+}
+
+// Returns an array of binding components for
+// {@link Observable#bind} from a set of iterable observables.
+//
+// @param {Iterable.<Observable>} observables
+// @param {String} attribute
+// @returns {Array.<String>}
+function getBindingTargets( observables, attribute ) {
+	return Array.prototype.concat( ...observables.map( observable => [ observable, attribute ] ) );
```

Comment:
Can't it be just that:
```js
return observables.map( observable => [ observable, attribute ] );
```

Reply:
It seems so. I admit that I haven't even looked at the code ;)

Reply:
Ah, no. It needs to be a `concat()` call. Because it takes `[1,2],[3,4],...` arrays and returns `[1,2,3,4...]`. Am I right, @jodator?

Reply:
Hmm, after short analyzing I've found that it might be correct as the result of this fn is `[ observable1, attribute, observable2, attribute ... ]`.

But docs are incorrect here anyway, `observables` can't be here iterable and the return type is different.

Reply:
Feel free to fix :)

Reply:
Yeah - return type should be `Array.<Observable|String>` as the result is as @ma2ciek wrote.

Reply:
Ok, I'll change it.